src/sentry/testutils/cases.py

```diff
@@ -2943,9 +2943,6 @@ def build_and_store_session(
 
 
 class OrganizationMetricsIntegrationTestCase(MetricsAPIBaseTestCase):
-    def __indexer_record(self, org_id: int, value: str) -> int:
```

Comment:
Removed because it was unused and caused confusion to the children of the class.